es-design-system/pages/examples/form-with-validation.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+    <b-container>
+        <b-row>
+            <b-col>
+                <h1>
+                    Form with Validation
+                </h1>
+                <h2>
+                    UX Guidelines
+                </h2>
+                <p>
+                    <ul>
+                        <li>
+                            Validation should initially trigger <em>on-blur</em>.<br>
+                            Subsequent validation should trigger on <em>input-change</em>.
+                        </li>
+                        <li>
+                            Validation for password fields should always trigger on <em>input-change</em>
+                        </li>
```

Comment:
For consistency, I might suggest a slight copy tweak here to either always have "on" inside the `<em>` or always outside - and I'd suggest removing the hyphenation since, as written, I don't think these refer to specific event names beyond the single words `blur` and `change`.

As an aside - I found on HPM that dropdown selects can't validate on blur, since `<b-dropdown>` doesn't fire a `blur` event - which is mostly fine, since you pretty much only need `required` validation on dropdowns anyway, so once we have an EsFormSelect component, we may need to update this guidance slightly to include a variance for dropdowns.

Reply:
Copy changes via [54b0ac8](https://github.com/EnergySage/es-ds/pull/393/commits/54b0ac8146c3591c8deaceeb5f7536a52c5ac98b)

> once we have an EsFormSelect component, we may need to update this guidance slightly to include a variance for dropdowns

Agreed 👍